libs/ledger-live-common/package.json

```
@@ -121,11 +121,11 @@
   "dependencies": {
     "@blooo/hw-app-acre": "^1.0.1",
     "@cardano-foundation/ledgerjs-hw-app-cardano": "^7.1.2",
-    "@celo/connect": "^3.0.1",
-    "@celo/contractkit": "^3.0.1",
-    "@celo/utils": "^3.0.1",
-    "@celo/wallet-base": "^3.0.1",
-    "@celo/wallet-ledger": "^3.0.1",
+    "@celo/connect": "^6.0.2",
+    "@celo/contractkit": "^8.3.0",
+    "@celo/utils": "^7.0.0",
+    "@celo/wallet-base": "^6.0.2",
+    "@celo/wallet-ledger": "^6.0.2",
```

```suggestion
    "@celo/connect": "^6.0.3",
    "@celo/contractkit": "^8.3.1",
    "@celo/utils": "^7.0.0",
    "@celo/wallet-base": "^6.0.3",
    "@celo/wallet-ledger": "^6.0.3",
```


Bumping these to make sure the changes work with the latest blooo app and corrected v values